playwright/tests/keywordSearch.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { Given, Then, When } from "@cucumber/cucumber";
+import { expect } from "@playwright/test";
+import { CustomWorld } from "../support/setup";
+
+Given(
+  "I am on the DRB home page for searching",
+  async function (this: CustomWorld) {
+    return await this.page.goto(
+      "https://digital-research-books-beta.nypl.org/"
+    );
+  }
+);
+
+When("I input a search term", async function (this: CustomWorld) {
+  await this.page.locator("#searchbar-textinput-search-bar").fill("IBM 1401");
+  await this.page.locator("#searchbar-button-search-bar").click();
+});
+
+Then(
+  "I count how many titles have my keyword",
+  async function (this: CustomWorld) {
+    await this.page
+      .locator("//h1[contains(text(), 'Digital Research Books')]")
+      .waitFor();
+    const textIWant = this.page.locator("//h2/a[contains(text(),'IBM 1401')]");
```

Comment:
I would consider changing this to `this.page.locator("a", { hasText: "IBM 1401" });` because then the test won't break if the `h2` changes to an `h3` or another element at some point in the future.

Reply:
done.